### DISCUSSION TITLE
Vertical Lines During Roast

### DISCUSSION THREAD

**Discussion by 3dflyer87** [artisan-roaster-scope/artisan#1752] [Q&A]
Hello! 

I'm wondering if anyone knows what the second vertical line references on the roast display. Seems like it should be a projected estimate for an event like first crack, but I'm not quite sure. 
Any insight appreciated! 
![Screenshot_20241125-224301_Google](https://github.com/user-attachments/assets/113fcd11-1951-4365-bb6a-2462f313b59f)

**Comment by MAKOMO** [comment]
The [AUC guide line](https://artisan-roasterscope.blogspot.com/2016/11/area-under-curve-auc.html)

**Reply by 3dflyer87** [reply]
Thank you!
